praw/reddit.py

```diff
@@ -389,7 +389,6 @@ def _prepare_objector(self):
             "moderators": models.ModeratorsWidget,
             "more": models.MoreComments,
             "post-flair": models.PostFlairWidget,
-            "stylesheet": models.Stylesheet,
```

Comment:
Why don't we update the mapper to use the new object?

Reply:
The new object requires a Subreddit to be initialized, so the logic to return a new instance would require initialization of a Subreddit class to bind the data to. Furthermore, there's only one method that returns stylesheet data, so there's no need to put it into an objector.